snapcraft.yaml

```diff
@@ -0,0 +1,81 @@
+name: couchdb
+version: 2.0
+summary: Document based database
+description: CouchDB is a database that completely embraces the web. Store your data with JSON documents. Access your documents and query your indexes with your web browser, via HTTP. Index, combine, and transform your documents with JavaScript. 
+confinement: strict
+grade: stable
+
+apps:
+    server:
+        daemon: simple
+        command: rel/couchdb/bin/snap_run
+        plugs: [network-bind, process-control, mount-observe]
+    couchdb:
+        command: rel/couchdb/bin/snap_run
+        plugs: [network-bind]
+parts:
+    couchdb: 
+        plugin: make
+        source: .
+        prepare: |
+          ./configure --disable-docs
+        build: |
+          make release
+          cp -ra ./rel $SNAPCRAFT_PART_INSTALL
+          cp -ra ./bin $SNAPCRAFT_PART_INSTALL
+        build-packages:
+          - binutils
+          - libc6-dev
+          - gcc
+          - g++
+          - make
+          - erlang-dev
+          - erlang-base-hipe
+          - erlang-reltool
+          - libcurl4-openssl-dev
+          - libmozjs185-dev
+          - libnspr4-dev
+          - libicu-dev
+          - icu-devtools
+          # For erlang/rebar processing
+          - erlang-nox
+          - erlang-os-mon
+          - erlang-syntax-tools
+          # For fauxton
+          - nodejs-dev
+          - nodejs-legacy
```

Comment:
nodejs-legacy is needed for the 'node' executable, which IIRC is being called from somewhere in the build process